examples/tests/README.md

```diff
@@ -0,0 +1,46 @@
+# Testing SBOMs Collector
+
+This script aims to fetch metadata files from high-quality GitHub repositories written in various programming languages and generate SBOMs (Software Bill of Materials) in three different formats: spdx-json, spdx, and cyclonedx.
+
+## Workflow
+
+The script follows the following workflow:
+
+1. Downloads the `README.md` file from the appropriate `awesome-XXX` repository for a specific programming language. These repositories are curated collections of high-quality repositories contributed by other users.
+
+2. Extracts the URLs of high-quality repositories from the `awesome-XXX` repository.
+
+3. Searches the corresponding repositories and verifies if they contain the desired metadata files. If the files are found, they are downloaded to the designated folder in the `./metadata/` directory.
+
+4. Utilizes trivy to generate three types of SBOMs (spdx-json, spdx, and cyclonedx) using the downloaded metadata files. The resulting SBOMs are saved in the `./SBOM/` directory.
+
+**Note:** Before executing the script, make sure to replace the `ACCESS_TOKEN` with your own GitHub access token.
+
+## Getting Started
+
+To get started with this script, follow the instructions below:
+
+1. Replace the `ACCESS_TOKEN` in the script with your GitHub access token. Make sure to grant the necessary permissions for accessing repositories and metadata.
+
+2. Run the script using the following command:
+
+```bash
+python download_metadata.py
+```
+
+3. Sit back and relax while the script downloads the metadata files and generates the SBOMs.
+
+## File Structure
+
+The repository has the following structure:
+
+```
+├── fetch_metadata.js     # Main script file
+├── metadata/             # Folder to store downloaded metadata files
```

Comment:
I noticed there are ZIP files in this PR but but not folders? Can you please help me understand? Are they the same thing really?

Reply:
In my latest push, I deleted two folders containing numerous files and added two zip files. I will update the README.md